packages/flutter/lib/src/widgets/fade_in_image.dart

```diff
@@ -16,6 +16,14 @@ import 'transitions.dart';
 // Examples can assume:
 // Uint8List bytes;
 
+// Same as in painting/image.dart.
```

Comment:
Can we put this in a common place and reuse it?

Reply:
Maybe it could be a static method on ImageProvider?

Reply:
Moved to a static on ResizeImage

Reply:
> Maybe it could be a static method on ImageProvider?

Will this PR merge to the current last stable version v1.9.1+hotfix.6？